package.json

```diff
@@ -23,5 +23,16 @@
   },
   "bugs": {
     "url": "https://github.com/Pomax/react-onoutsideevent/issues"
+  },
+  "devDependencies": {
+    "envify": "^3.4.0",
+    "gulp": "^3.9.0",
+    "gulp-rename": "^1.2.2",
+    "gulp-uglify": "^1.4.2",
+    "vinyl-transform": "^1.0.0"
+  },
+  "scripts": {
+    "build": "./node_modules/.bin/gulp",
```

Comment:
Why would dist not be present? It's committed and not removed when you npm run clean.